server/routes/index.js

```diff
@@ -12,4 +13,7 @@ module.exports = (app) => {
 
   // drills
   app.post('/api/sessions/:session_id/drills', drillsController.create);
+
+  app.get('/api/actions/all', actionsController.GetAll);
```

Comment:
noun should be the last in the api. so for getting all the actions it should be api/actions/. For an individual action it should be api/action_id/actions